myuw/views/api/emaillist.py

```diff
@@ -21,21 +26,44 @@ def GET(self, request, year, quarter,
         """
         timer = Timer()
         try:
-            email_list_json = get_section_email_lists(
-                get_section(year, quarter,
-                            curriculum_abbr,
-                            course_number,
-                            section_id), True)
+            if not is_instructor(request):
```

Comment:
This check validates the URL params, but then email lists are created based on get_input.  It looks like someone can bypass the security test here by posting wrong data to the right url